### PR TITLE
Add create, swap and linked transfer methods

### LINKED ISSUE #50: Support other methods for Zympay
```
0. Create
1. Deposit
2. Swap
3. ConditionalTransfer (linked transfer)
4. ResolveCondition
5. Withdraw
6. GetFreeBalance
```

### PR DESCRIPTION
**Pull request by pedrouid**
Fix #50

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -152,6 +197,7 @@ export default class ClientManager {
 
   public async setMnemonic(mnemonic: string) {
     await storeMnemonic(mnemonic, this._store);
+    console.log("resolved to store mnemonic");
```
Comment (ghost) [inline src/client.ts:200]: console.log?

File: src/helpers/store.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { EventSubscription, InitOptions } from "./types";
 import { IStoreService } from "@connext/types";
 
 export function storeMnemonic(mnemonic: string, store: IStoreService): Promise<void> {
+  console.log("STORE MNEMONIC", mnemonic);
```
Comment (ghost) [inline src/helpers/store.ts:10]: console.log?

**Comment by pedrouid** [comment]
Replaced by #54

File: src/client.ts (hunk 1 above)
Comment (pedrouid) [reply src/client.ts:200]: My bad, that was a typo

File: src/helpers/store.ts (hunk 2 above)
Comment (pedrouid) [reply src/helpers/store.ts:10]: same as above

**Pull request closed without merging**